### PR TITLE
Remove pkg/util package

### PR DESCRIPTION
**Pull request by invidian**
**Includes commits from #951**

Part of #290

### CODE REVIEW

**Review by surajssd — DISMISSED** [review]
LGTM

File: pkg/lokomotive/cluster.go [hunk 1]
```diff
@@ -20,12 +20,25 @@ import (
...
 
 	v1 "k8s.io/api/core/v1"
-	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
 )
 
+const (
+	// Max number of retries when waiting for cluster to become available.
```
Comment (knrt10) [inline pkg/lokomotive/cluster.go:32]: I think this is not a number anymore, this comment should be changed. What do you think?

Comment (invidian) [reply pkg/lokomotive/cluster.go:32]: Yes, thanks.

**Comment by surajssd** [comment]
I have noticed that 6minutes looks like a good time for the nodes to join. So we should timeout after 6minutes. See https://github.com/kinvolk/lokomotive/issues/940#issuecomment-692618588 for more details.

**Comment by invidian** [comment]
>I have noticed that 6minutes looks like a good time for the nodes to join. So we should timeout after 6minutes. See #940 (comment) for more details.

Right, let me check it. What about some nice number like 5 minutes for API and 10 minutes for nodes?

**Review by knrt10 — DISMISSED** [review]
+1

**Review by knrt10 — DISMISSED** [review]

**Review by surajssd — DISMISSED** [review]

**Comment by surajssd** [comment]
AKS didn't start on this :-\

**Comment by knrt10** [comment]
That's weird because no other process is running.

**Comment by invidian** [comment]
Rebased and push again to poke CI.

**Review by surajssd — APPROVED** [review]

**Review by knrt10 — APPROVED** [review]

**Pull request merged**